src/opm/input/eclipse/share/keywords/001_Eclipse300/C/CNAMES

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "CNAMES",
+  "sections": [
+    "PROPS"
+  ],
+  "size": "UNKNOWN",
```

Comment:
For what it's worth we usually try to avoid adding new keywords with an unknown size.  Is there no other way to achieve your goal?  How about using `"data"` instead of `"items"`? I.e., something along the lines of
```json
{
  "name" : "CNAMES",
  "sections" : [
    "PROPS"
  ],
  "data" : {
    "value_type": "STRING"
  }
}
```
That would push a little bit of consistency checking, especially the number of strings viz `NCOMPS` value, to higher level logic and you'd be restricted to using indexed instead of named access, but I do think it'd be a bit more flexible.

Reply:
Thank you for the comment, will modify all relevant keywords accordingly.